projects/ama-sdk/src/lib/helpers/common.ts

```diff
@@ -39,3 +39,12 @@ export interface EntityDialogPayload {
     values?: EntityDialogForm;
     action: any;
 }
+
+export const sanitizeString = (text: string) => {
+    const allowedChars = 'a-zA-Z0-9_';
```

Comment:
This regex is used either in the directive and here. Please extract it into create-entries-names.ts file into a constant named MODEL_NAME_CHARACTERS, from where we can reuse it, and have it in only one place, so in case of needing of modification, we don't need to hunt down the locations about where to modify these values.